modules/ztp-filtering-ai-crs-using-siteconfig.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-filtering-ai-crs-using-siteconfig_{context}"]
+= Filtering custom resources using SiteConfig filters
+
+You can configure `SiteConfig` custom resources (CRs) to include or exclude CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline.
+
+You define a set of extra manifests for inclusion in the cluster install. These manifests are linked to the `SiteConfig` custom resources (CRs) and are applied to the cluster during installation. Using filters, you can easily customize individual `SiteConfig` CRs based on your requirements.
+
+The following example configures a custom single-node cluster `SiteConfig` CR that excludes `extra-manifest/03-sctp-machine-config-worker.yaml` and instead applies a custom `extra-manifest/custom-sctp-machine-config-worker.yaml` file in the install.
+
+.Prerequisites
+
+. Create a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create a set of extra manifest CRs that the ZTP pipeline uses to customize the cluster installs.
+
+. Change to your custom `/siteconfig` directory. The following example illustrates a sample `/siteconfig` folder:
++
+[source,text]
+----
+siteconfig
+├── site1-sno-du.yaml
+└── extra-manifest
+    └── 03-sctp-machine-config-worker.yaml
+    └── custom-sctp-machine-config-worker.yaml
+----
+
+. Edit the custom `site1-sno-du.yaml` file to set the `exclude` and `include` files, for example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "site1-sno-du"
+  namespace: "site1-sno-du"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "<ssh_public_key>"
+  clusters:
+  - clusterName: "site1-sno-du"
+    networkType: "OVNKubernetes"
+    clusterLabels:
+      sites : "site1-sno-du"
+
+    ...
+
+  extraManifestPath: extra-manifest <1>
```

Comment:
Perhaps a different name than extra-manifest would be ideal. I'm not sure if this would cause any problem when the user manifests dir is having the same name as ztp source-crs/manifest-dir. But it may cause confusion even if it will be properly handled.

Another small thing is qe has been using "my-manifests-dir/" format as described in README. Not sure if "my-manifests-dir" would also work (please confirm with SME).